index.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
         'order/properties-order': [
             {
                 // Must be first.
-                properties: ['all'],
+                properties: ['all', 'appearance'],
```

Comment:
I'm not sure about having `appearance` grouped with `all`. It looks like Bootstrap have [added the property](https://github.com/twbs/stylelint-config-twbs-bootstrap/blob/main/css/index.js), but they've grouped it with `transform` and `animation` properties which I'm not sure makes total sense either. 

I think having it in the "Accessibility and Interactions" group might be best — makes the most sense to me for grouping, and it's fairly close to where Bootstrap has it so compatibility isn't thrown off too much.

Reply:
This property changed almost *all* the style of element, for example, `appearance: button` let `<a>` looks like a `<buttom>`, so as my opinion, `all` and `appearance` have a lot in common.

I don't know why Bootstrap grouped it with `transform`, which confuses me. If this is a wrong thing, should we maintain compatibility with it?

Of course, the decision is yours 🤣

Reply:
> I'm not sure about having `appearance` grouped with `all`. It looks like Bootstrap have [added the property](https://github.com/twbs/stylelint-config-twbs-bootstrap/blob/main/css/index.js), but they've grouped it with `transform` and `animation` properties which I'm not sure makes total sense either. 
> 
> I think having it in the "Accessibility and Interactions" group might be best — makes the most sense to me for grouping, and it's fairly close to where Bootstrap has it so compatibility isn't thrown off too much.

Appearance is about changing default style so it should be right on top, before any custom styles come in. Has it been already implemented with another PR?